src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -125,8 +143,34 @@ public static unsafe void PrepareMethod(RuntimeMethodHandle method, RuntimeTypeH
         [MethodImpl(MethodImplOptions.InternalCall)]
         public static extern void PrepareDelegate(Delegate d);
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static int GetHashCode(object? o)
```

Comment:
You could apply a similar treatment to `TryGetHashCode` below.

Reply:
Thanks! Applied